mods/ts/rules/nod-structures.yaml

```diff
@@ -267,6 +271,8 @@ NAHPAD:
 		IsPlayerPalette: false
 		LineWidth: 2
 		ForceSetType: Air
+	CommandBarBlacklist:
+		DisableStop: false		
```

Comment:
There seems to be 2 tabs after false in most of these. Only RA structures.yaml has it correct.

Also the description of the Stop button should be updated to say it resets the rally points.

Reply:
Heads up for spotting those two tabs. They shouldn't be there and I corrected that!
I'm looking into changing the description of the button - you're spot on with that!
Thanks for the review!